src/services/item/fluent-schema.ts

```diff
@@ -159,8 +160,17 @@ export const getParents = {
 };
 
 export const getOwn = {
+  querystring: S.object()
+    .additionalProperties(false)
+    .prop('page', S.number().default(1))
+    .prop('name', S.string())
+    .prop('limit', S.number().default(ITEMS_LIST_LIMIT))
+    .prop('all', S.boolean().default(false)),
```

Comment:
I think it's maybe too much to add `all` just to disable pagination. What about setting a big limit instead? (sorry I think I told the opposite last time).

Reply:
@pyphilia 
Thanks for your comments, really appreciated!
Do you mean to send a limit as a total count or a big number instead of having all as a flag?